api/controller/trigger.go

```diff
@@ -159,6 +161,10 @@ func DeleteTriggerThrottling(database moira.Database, triggerID string) *api.Err
 
 // SetTriggerMaintenance sets maintenance to metrics and whole trigger
 func SetTriggerMaintenance(database moira.Database, triggerID string, triggerMaintenance dto.TriggerMaintenance, userLogin string, timeCallMaintenance int64) *api.ErrorResponse {
+	if err := database.AcquireTriggerCheckLock(triggerID, maxTriggerLockAttempts); err != nil {
+		return api.ErrorInternalServer(err)
+	}
+	defer database.DeleteTriggerCheckLock(triggerID) //nolint
```

Comment:
This piece of code was copied from another trigger lock usage and I think this was done in that way by design. The lock has expiration time 30 sec and it assumes that even fail on deleting lock after succesful operation should not prevent successful response. Would you think it will make more sense to log that error?

Reply:
> This piece of code was copied from another trigger lock usage

I know 😀

> Would you think it will make more sense to log that error?

Can you suggest another way not to write `nolint`?

Reply:
Sure, actually I already did it. We can handle the error by logging it
> Would you think it will make more sense to log that error?

Reply:
> log that error

good idea

Reply:
Updated